docs/src/software.md

```diff
@@ -67,18 +67,19 @@ The [Dockerfile](https://github.com/ICTU/quality-time/blob/master/components/ext
 
 The external server uses the following environment variables:
 
-| Name | Default value | Description                                                                                                                                                                                                                                                                 |
-| :--- | :------------ |:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `EXTERNAL_SERVER_PORT` | `5001` | Port of the external server.                                                                                                                                                                                                                                         |
-| `DATABASE_URL` | `mongodb://root:root@database:27017` | Mongo database connection URL.                                                                                                                                                                                                                 |
-| `LDAP_URL` | `ldap://ldap:389` | Comma-separated list of LDAP connection URL(s).                                                                                                                                                                                                                       |
-| `LDAP_ROOT_DN` | `dc=example,dc=org` | LDAP root distinguished name.                                                                                                                                                                                                                                   |
-| `LDAP_LOOKUP_USER_DN` | `cn=admin,dc=example,dc=org` | LDAP lookup user distinguished name.                                                                                                                                                                                                            |
-| `LDAP_LOOKUP_USER_PASSWORD` | `admin` | LDAP lookup user password.                                                                                                                                                                                                                                     |
+| Name | Default value | Description |
+| :--- | :------------ |:------------|
+| `EXTERNAL_SERVER_PORT` | `5001` | Port of the external server. |
+| `EXTERNAL_SERVER_LOG_LEVEL` | `WARNING` | Log level. Allowed values are `DEBUG`, `INFO`, `WARNING`, `CRITICAL`, and `ERROR`. |
```

Comment:
The order of critical and error should be switched

Reply:
Good catch!